### PR TITLE
splash page event tags

### PR DESCRIPTION
**Pull request by duanwilliam**
oops stanley wanted me to (push) this and i forgot to do it earlier
the event tags stuff i have its probably scuffed as heck

### CODE REVIEW

**Comment by duanwilliam** [comment]
@brilam8 @garrettluu

File: src/database/models/eventsModel.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-var mongoose = require('mongoose');
-var Schema = mongoose.Schema;
+let mongoose = require('mongoose');
+let Schema = mongoose.Schema;
 
 //Event model consisting of the event title, description, start and end time, checkIn code, and splash URL
-var eventSchema = new Schema(
+let eventSchema = new Schema(
```
Comment (leestanley) [inline src/database/models/eventsModel.js:5]: Good call to switch away from let - why don't we just switch all our variable types to const?

File: src/routes/eventRouter.js [hunk 2]
```diff
@@ -23,23 +23,33 @@ routes.post('/event', (request, res) => {
...
+      res.status(200).send("deleted");
+    } catch (err) {
+      console.log('error ', err);
+    }
+});
+
 // GET specific events from the database via a "query" (title search, description, time, etc.)
 routes.get("/event/:query", (request, res) => {
-    var event = Event.find({ $text: { $search: request.params.query}}, function(err, events){
+    let event = Event.find({ $text: { $search: request.params.query}}, function(err, events){
```
Comment (leestanley) [inline src/routes/eventRouter.js:52]: Can just be a const?

File: src/routes/eventRouter.js [hunk 3]
```diff
@@ -23,23 +23,33 @@ routes.post('/event', (request, res) => {
...
 
+routes.delete('/event', (request, res) => {
+    try {
+      Event.findByIdAndDelete(request.body.id);
+      res.status(200).send("deleted");
+    } catch (err) {
+      console.log('error ', err);
+    }
+});
+
```
Comment (leestanley) [inline src/routes/eventRouter.js:49]: We have merge conflicts for the lines above/already implemented so we could probably take these out

**Pull request closed without merging**